### PR TITLE
Add set multi term on runtime

### PR DESCRIPTION
**Pull request by schonhoff**
This pull request adds the possibility to change the multi_term search on runtime.

See https://github.com/yajra/laravel-datatables/issues/2575

### CODE REVIEW

**Comment by schonhoff** [comment]
@yajra Should I include a test for this or is it not necessary? And can you review and add this in the upcoming version? Would be really nice!

File: src/DataTableAbstract.php [hunk 1]
```diff
@@ -507,6 +507,19 @@ public function startsWithSearch($state = true)
         return $this;
     }
 
+    /**
+     * Set multi_term search config at runtime.
+     *
+     * @param bool $multiTerm
+     * @return $this
+     */
+    public function setMultiTerm(bool $multiTerm = true)
```
Comment (yajra) [inline src/DataTableAbstract.php:516]: Maybe we should remove the `bool` type hint just to be sure this does not break on lower versions?

Comment (schonhoff) [reply src/DataTableAbstract.php]: Added a commit without the type hint, so it won't break anything in lower versions. bool was included on php 7 as a type hint so maybe someone is using it on 5.x still :)

**Comment by yajra** [comment]
Released on v9.16.0, thanks! 🚀

**Pull request merged**